pkg/cluster/constants.go

```diff
@@ -3,11 +3,12 @@ package cluster
 import "k8s.io/apimachinery/pkg/labels"
 
 const (
-	RookCephNS                   = "rook-ceph"
-	CephClusterName              = "rook-ceph"
-	RookCephSharedFSMetadataPool = "rook-shared-fs-metadata"
-	RookCephSharedFSDataPool     = "rook-shared-fs-data0"
-	CephDeviceHealthMetricsPool  = "device_health_metrics"
+	RookCephNS                        = "rook-ceph"
```

Comment:
```suggestion
	rookCephNS                        = "rook-ceph"
```
Do we use this const outside of this package?
If not, do we really need to export them?

Reply:
i'd rather not change this as there are many exported constants in this file that likely do not need to be exported for the same reason

Reply:
I think we could change all right? We do not need to export anything from here.
Also, all that is exported need to have a comment/doc otherwise when we be able to add the lint here it will fail.

Reply:
Possibly but its out of scope of this change